source/observables.txt

```diff
@@ -0,0 +1,243 @@
+.. _scala-observables:
+
+==============================
+Access Data From an Observable
+==============================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, subscribe, non-blocking
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to access the results of MongoDB operations
+from an ``Observable`` instance.
+
+An ``Observable`` represents a stream of data emitted by an operation
+over time. To access this data, you can create an ``Observer`` instance
+that subscribes to the ``Observable``. 
+
+How to Process an Observable
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To run a MongoDB operation and process its data, you must request
+the operation results from an ``Observable``. The ``Observable``
+is a higher-level abstraction built on top of a cursor, or a 
+mechanism that fetches results in batches. The {+driver-short+}
+handles the cursor internally and exposes the operation
+results through an observable interface.
+
+Each operation returns an instance of the ``Observable[T]`` type, where the ``[T]``
+parameterization corresponds to the type of each item stored by the
+observable. Some operations return a specialized observable that
+extends the general ``Observable[T]`` type and provides additional
+methods to filter and process results. These specialized observables
+include the following types:
+
+- ``SingleObservable[T]``: Returned by operations that produce a single result
+- ``FindObservable[T]``: Returned by the ``find()`` method
+- ``DistinctObservable[T]``: Returned by the ``distinct()`` method
+- ``AggregateObservable[T]``: Returned by the ``aggregate()`` method
+
+.. tip::
+
+  To view a full list of ``Observable`` types, see the `Observable <{+api+}/org/mongodb/scala/Observable.html>`__
+  API documentation.
```

Comment:
Nit: We link to the Observable documentation a few other times on this page – can we remove this admonition?